pages/contact.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./../css/contact.css" />
+    <link rel="stylesheet" href="./../css/style.css" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+    />
+    <title>Contact - Hostal las paisas</title>
+  </head>
+  <body>
+    <nav>
+      <!-- nav bar section -->
+    </nav>
+    <section class="small-hero small-hero__image--contact">
+      <div class="small-hero__layer"></div>
```

Comment:
Sorry for not being clear in the design with this. We need to include an extra layer for getting this darker. Please add another div with class `small-hero__layer--darker` with a black background color with `0.3` of opacity. This needs to be positioned on top of the current `small-hero__layer`.

Reply:
This can be implemented by @Sonny97 since he is working in the home hero.